grin/src/Reducer/PrimOps.hs

```diff
@@ -32,7 +32,7 @@ C.include "<math.h>"
 C.include "<stdio.h>"
 
 -- primitive functions
-primLiteralPrint _ _ [RT_Lit (LInt64 a)] = liftIO (print a) >> pure RT_Unit
+primLiteralPrint _ _ [RT_Lit (LInt64 a)] = liftIO (putStr $ show a) >> pure RT_Unit
```

Comment:
Is this better?

Reply:
It doesn't print out an extra newline. This is a fix for some test.